### PR TITLE
Dfpl 2159 Allocation Proposal playwright Test

### PR DESCRIPTION
**Pull request by Braimah101**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DFPL-2159

### Change description ###
As part of the FPL, we need to create a  Allocation proposal to the new Playwright framework. This task involves migrating the test suite and updating any necessary configurations and tests as required.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

**Review by Iphiclus — CHANGES_REQUESTED** [review]

File: playwright-e2e/tests/create-case.spec.ts [hunk 1]
```diff
@@ -4,14 +4,16 @@ import { newSwanseaLocalAuthorityUserOne } from "../settings/userCredentials";
 import { CreateCase } from "../pages/create-case";
 import { StartApplication } from "../pages/start-application";
 import { OrdersAndDirectionSought } from "../pages/orders-and-directions";
+import {AllocationProposal} from "../pages/allocation-proposal";
 
 // WIP
-test.skip("Create a case", async ({ page }) => {
+test("Create a case", async ({ page }) => {
```
[the remark this answers is not in the document]
Comment (Braimah101) [reply playwright-e2e/tests/create-case.spec.ts:10]: True

File: service/.properties [hunk 2]
```diff
@@ -0,0 +1 @@
+buildDir=../build
```
[the remark this answers is not in the document]
Comment (Braimah101) [reply service/.properties]: True

File: playwright-e2e/fixtures/create-fixture.ts [hunk 3]
```diff
@@ -6,6 +6,7 @@ import { OrdersAndDirectionSought } from "../pages/orders-and-directions";
 import { HearingUrgency } from "../pages/hearing-urgency";
 import { GroundsForTheApplication } from "../pages/grounds-for-the-application";
 import { RiskAndHarmToChildren } from "../pages/risk-and-harm-to-children";
+import {AllocationProposal} from "../pages/allocation-proposal";
```
[the remark this answers is not in the document]
Comment (Braimah101) [reply playwright-e2e/fixtures/create-fixture.ts]: True

File: playwright-e2e/tests/smoke-test.spec.ts [hunk 4]
```diff
@@ -43,13 +45,21 @@ test("Smoke Test @smoke-test", async ({
...
 
-  // 5. Grounds for the application
+  // // 5. Grounds for the application
   await startApplication.groundsForTheApplication();
   await groundsForTheApplication.groundsForTheApplicationHeading.isVisible();
   await groundsForTheApplication.groundsForTheApplicationSmokeTest();
   await startApplication.groundsForTheApplicationHasBeenUpdated();
 
-  // 6. Risk and harm to children
+  // // 6. Risk and harm to children
```
[the remark this answers is not in the document]
Comment (Braimah101) [reply playwright-e2e/tests/smoke-test.spec.ts]: True

File: playwright-e2e/tests/smoke-test.spec.ts [hunk 5]
```diff
@@ -43,13 +45,21 @@ test("Smoke Test @smoke-test", async ({
   await hearingUrgency.saveAndContinueButton.click();
   await startApplication.addApplicationDetailsHeading.isVisible();
 
-  // 5. Grounds for the application
+  // // 5. Grounds for the application
```
[the remark this answers is not in the document]
Comment (Braimah101) [reply playwright-e2e/tests/smoke-test.spec.ts]: True

File: service/.properties [hunk 6]
```diff
@@ -0,0 +1 @@
+
```
Comment (Iphiclus) [inline service/.properties:1]: Hi @Braimah101 I think this file needs to be untracked and removed from your changes.

File: playwright-e2e/test-2.spec.ts [hunk 7]
```diff
@@ -0,0 +1,33 @@
+import { test, expect } from '@playwright/test';
```
Comment (Iphiclus) [inline playwright-e2e/test-2.spec.ts:1]: Hi @Braimah101 is this file needed?

File: playwright-e2e/tests/create-case.spec.ts [hunk 8]
```diff
@@ -4,14 +4,16 @@ import { newSwanseaLocalAuthorityUserOne } from "../settings/userCredentials";
 import { CreateCase } from "../pages/create-case";
 import { StartApplication } from "../pages/start-application";
 import { OrdersAndDirectionSought } from "../pages/orders-and-directions";
+import {AllocationProposal} from "../pages/allocation-proposal";
```
Comment (Iphiclus) [inline playwright-e2e/tests/create-case.spec.ts:7]: Hi @Braimah101 {AllocationProposal} should be { AllocationProposal } (spaces)

File: playwright-e2e/tests/create-case.spec.ts [hunk 9]
```diff
@@ -4,14 +4,16 @@ import { newSwanseaLocalAuthorityUserOne } from "../settings/userCredentials";
...
 
 // WIP
-test.skip("Create a case", async ({ page }) => {
```
Comment (Iphiclus) [inline playwright-e2e/tests/create-case.spec.ts:9]: Hi @Braimah101 please fix back to test.skip

**Review by Iphiclus — CHANGES_REQUESTED** [review]

File: playwright-e2e/tests/create-case.spec.ts (hunk 8 above)
Comment (Braimah101) [reply playwright-e2e/tests/create-case.spec.ts:7]: True

File: e2e/helpers/hooks_helper.js [hunk 10]
```diff
@@ -1,5 +1,5 @@
 const recorder = require('codeceptjs').recorder;
-const output = require('codeceptjs').output;
+//const output = require('codeceptjs').output;
```
Comment (GauravChawlaMOJ) [inline e2e/helpers/hooks_helper.js:2]: may be delete it rather than commenting out?

File: playwright-e2e/tests/create-case.spec.ts [hunk 11]
```diff
@@ -20,47 +22,14 @@ test.skip("Create a case", async ({ page }) => {
...
   await createCase.createCase();
   await createCase.submitCase(createCase.generatedCaseName);
   await createCase.checkCaseIsCreated(createCase.generatedCaseName);
 
-  // 3. Orders and directions sought
+  // Orders and directions sought
   await startApplication.ordersAndDirectionsSoughtLink.isVisible();
   await startApplication.ordersAndDirectionsSoughtLink.click();
   await ordersAndDirectionSought.OrdersAndDirectionsHeading.isVisible();
-
```
Comment (GauravChawlaMOJ) [inline playwright-e2e/tests/create-case.spec.ts:33]: may be keep these comments for future implementations?

File: playwright.config.ts [hunk 12]
```diff
@@ -14,8 +14,8 @@ export default defineConfig({
   testDir: "./playwright-e2e",
   /* Run tests in files in parallel */
   fullyParallel: true,
-  timeout: 180000,
-  expect: { timeout: 180000 },
+  timeout: 250000,
```
Comment (GauravChawlaMOJ) [inline playwright.config.ts:17]: is timeout needed to be raised due to allocation proposal screens only?

File: playwright-e2e/tests/create-case.spec.ts (hunk 11 above)
Comment (Iphiclus) [reply playwright-e2e/tests/create-case.spec.ts:33]: @GauravChawlaMOJ this test isn't being used, and we could delete it all together, I removed the comments as it was messy and would likely change anyway.

File: playwright.config.ts (hunk 12 above)
Comment (Iphiclus) [reply playwright.config.ts:17]: @GauravChawlaMOJ this was increased because Charles connection is so slow with AAT it can't handle less then this time out. It will likely help our nightly pipeline too as AAT is so problematic with performance.

**Review by GauravChawlaMOJ — DISMISSED** [review]

**Review by Iphiclus — APPROVED** [review]

**Review by GauravChawlaMOJ — APPROVED** [review]

File: playwright-e2e/tests/create-case.spec.ts (hunk 11 above)
Comment (Braimah101) [reply playwright-e2e/tests/create-case.spec.ts:33]: True
